### PR TITLE
Use rake task to install infrastructure files

### LINKED ISSUE #1547: Rake task to install infrastructure files
It would be handy now that there are 2 places to build packages (deb & RPM) to have a single rake task like `install:infrastructure:files` that handles installing things like sudo rules, crontab files, and other files that are needed during packaging.  The task would need to respect DESTDIR which is used for RPM and deb builds.

I could have sworn I saw a similar task to this in the past but a few searches turned up nothing.

### PR DESCRIPTION
**Pull request by treydock**
Fixes #1547
Also a bit of cleanup of unused rake task code

While I was dealing with install code, I moved main Rakefile logic into what I hope is more appropriate place.

### CODE REVIEW

File: lib/tasks/install_utils.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require_relative 'build_utils'
+require 'erb'
+require 'dotenv'
+require 'tempfile'
+
+module InstallUtils
+  include BuildUtils
```
Comment (johrstrom) [inline lib/tasks/install_utils.rb:9]: I think just 1 file is fine. It seems commonplace to have very large files in ruby right?

Name could change though. Happy to throw in helper or rake like `ood_rake_helper` or `ood_utilities`? :man_shrugging:

File: lib/tasks/install_utils.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      },
+      {
+        src: 'sudo.erb',
+        dest: File.join(DESTDIR, 'etc/sudoers.d/ood'),
+        mode: 0440,
+      },
+    ]
+  end
+
+  def package_file_render(name)
```
Comment (johrstrom) [inline lib/tasks/install_utils.rb:56]:
```suggestion
  def render_package_file(name)
```

File: lib/tasks/install_utils.rb (hunk 1 above)
Comment (johrstrom) [reply lib/tasks/install_utils.rb:9]: lol we already have `rake_helper.rb`.

Comment (treydock) [reply lib/tasks/install_utils.rb:9]: There is already a `rake_helper` file, could just put everything in there?

Comment (treydock) [reply lib/tasks/install_utils.rb:9]: Yep, your second comment didn't show up right away for me, I'll put into `rake_helper.rb`.

Comment (treydock) [reply lib/tasks/install_utils.rb]: This change was made, moved all helper methods into `rake_helper.rb`

**Review by johrstrom — APPROVED** [review]
I'm good with this when tests pass.

**Pull request merged**